src/github.com/getlantern/flashlight/config/fetcher.go

```diff
@@ -117,8 +118,9 @@ func (cf *fetcher) fetchCloudConfig(cfg *Config) ([]byte, error) {
 	req.Header.Set("Lantern-Fronted-URL", cfg.FrontedCloudConfig+cb)
 
 	id := cf.user.GetUserID()
-	if id != "" {
-		req.Header.Set(userIDHeader, id)
+	if id != 0 {
+		strId := strconv.FormatFloat(id, 'f', -1, 64)
+		req.Header.Set(userIDHeader, strId)
```

Comment:
This *could* send a formatted float like `12.34`, which is not expected by the server, the only thing that forbids that from happening is the program's logic.